### PR TITLE
Update Known-Issues-Priorities.md

### PR DESCRIPTION
**Pull request by 5chdn**

### CODE REVIEW

File: Known-Issues-Priorities.md [hunk 1]
```diff
@@ -20,12 +20,13 @@ Trying to prioritize stuff here. While bridging the gap to a stable light client
 - [ ] 5. [#7436](https://github.com/paritytech/parity/issues/7436) Look into why snapshotting fails on nodes with low memory. We won't be able to produce snapshots with OverlayRecent pruning; we should look into enabling a different pruning algorithm that does not require keeping recent states in memory.
 - [ ] 6. [#7177](https://github.com/paritytech/parity/issues/7177) Figure out why fast sync sometimes just stops in the middle. This is probably obsolete if we address the first 3 points in this list.
 - [ ] 7. [#7008](https://github.com/paritytech/parity/issues/7008) Figure out why ancient-blocks download sometimes stalls. Ancient blocks are important to serve the network as a full node.
-- [ ] 8. [#6956](https://github.com/paritytech/parity/issues/6956) Provide RPC health status indicator for ancient blocks in addition to sync status.
+- [ ] 8. [#8565](https://github.com/paritytech/parity/issues/8565) We want to investigate new snapshot formats.
```
Comment (Tbaut) [inline Known-Issues-Priorities.md]: Should the 4. be replaced by: https://github.com/paritytech/parity/pull/8544

Shouldn't the 8. replace the 3. [#7863](https://github.com/paritytech/parity/issues/7863) Fast-pruned nodes should be able to generate snapshots

File: Known-Issues-Priorities.md [hunk 2]
```diff
@@ -34,19 +35,19 @@ Trying to prioritize stuff here. While bridging the gap to a stable light client
 ### 3. Light Sync / Light Client
 
 - [ ] 1. [#6319](https://github.com/paritytech/parity/issues/6319) It's currently close to impossible to synchronize a light client. We need to figure out why this happens and fix this.
```
Comment (Tbaut) [inline Known-Issues-Priorities.md]: Not a prio anymore IMHO

File: Known-Issues-Priorities.md [hunk 3]
```diff
@@ -34,19 +35,19 @@ Trying to prioritize stuff here. While bridging the gap to a stable light client
...
-- [ ] 2. [#5642](https://github.com/paritytech/parity/issues/5642) Allow light-sync for Kovan/PoA networks.
+- [x] 2. [#5642](https://github.com/paritytech/parity/issues/5642) Allow light-sync for Kovan/PoA networks.
 - [ ] 3. [#6202](https://github.com/paritytech/parity/issues/6202) Enable RPCs for light clients similar to those available in a full node. This will be the ground-work for the light wallet (new UI).
 - [ ] 4. [#6010](https://github.com/paritytech/parity/issues/6010) Allow warp-syncing of light clients.
```
Comment (Tbaut) [inline Known-Issues-Priorities.md]: Not a prio anymore IMHO

File: Known-Issues-Priorities.md [hunk 4]
```diff
@@ -34,19 +35,19 @@ Trying to prioritize stuff here. While bridging the gap to a stable light client
...
 - [ ] 5. [#7263](https://github.com/paritytech/parity/issues/7263) Allow syncing the light client from a trusted block hash.
```
Comment (Tbaut) [inline Known-Issues-Priorities.md]: This is in 1.11 AFAIK, shouldn't we check it?

File: Known-Issues-Priorities.md (hunk 2 above)
Comment (5chdn) [reply Known-Issues-Priorities.md]: Introducing hard-coded sync is just a workaround, not a fix. Not being able to do a "full" light sync is the worst of all issues we have. IMHO. :)

Comment (Tbaut) [reply Known-Issues-Priorities.md]: Sure, I'd argue that other issues without any workaround should precede this one though.

File: Known-Issues-Priorities.md (hunk 1 above)
Comment (5chdn) [reply Known-Issues-Priorities.md]: Yes and no. 8 is different from 3.

File: Known-Issues-Priorities.md (hunk 2 above)
Comment (5chdn) [reply Known-Issues-Priorities.md]: For example?

File: Known-Issues-Priorities.md (hunk 3 above)
Comment (5chdn) [reply Known-Issues-Priorities.md]: Still a prio

**Pull request merged**